fabtests/common/shared.c

```diff
@@ -760,6 +761,14 @@ int ft_init_oob(void)
 	return ret;
 }
 
+/*
+ * This function is used to allow multiple client connections before quitting the
+ * server. The persistent server accepts the next client by posting a receive,
+ * exchanging its address with the next client, and insert the remote address into
+ * its av. It is expected that the server connects with one client at the
+ * same time, and the clients come and leave in sequence. See functional/rdm.c as
+ * an example of its usage.
+ */
```

Comment:
I would simplify this comment to describe the higher-level assumptions only.  Details like posting a receive, exchanging address, etc. can be determined directly from the code, and means if we change the code, we don't also need to remember to change the comment.  E.g.
```
Handles a persistent server communicating with multiple clients, one at a time, in sequence.
```

Reply:
Fixed in newest revision.